src/index.js

```diff
@@ -0,0 +1,8 @@
+api = require('./api/estados');
+
+const npmPackage = {
+  getCitiesFromState: api.getCities,
+  getStateFromCity: api.getStateFromCity
```

Comment:
Possible undefined variable: 'api' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)